File: src/main/java/com/baulsupp/oksocial/services/transferwise/TransferwiseUtil.java
package com.baulsupp.oksocial.services.transferwise;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class TransferwiseUtil {
  private TransferwiseUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet(
          "api.transferwise.com")
      );

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.transferwise.com" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/transferwise/TransferwiseUtil.kt
package com.baulsupp.oksocial.services.transferwise

import okhttp3.Request

object TransferwiseUtil {

    val API_HOSTS = setOf((
            "api.transferwise.com")
    )

    fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
        return requestBuilder.url("https://api.transferwise.com" + s).build()
    }
}
